amespahdbpythonsuite/mcfitted.py

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env python3
+
+from __future__ import annotations
+from typing import Optional
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from amespahdbpythonsuite.amespahdb import AmesPAHdb
+from amespahdbpythonsuite.spectrum import Spectrum
+
+message = AmesPAHdb.message
+
+
+class Mcfitted(Spectrum):
```

Comment:
'Mcfitted' -> 'MCFitted' and would not inherit from Spectrum. See the IDL implementation at https://github.com/PAHdb/AmesPAHdbIDLSuite/blob/master/src/amespahdbidlsuite_mcfitted_spectrum__define.pro.